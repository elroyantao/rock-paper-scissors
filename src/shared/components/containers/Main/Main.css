:global body, :global h3 {
  margin: 0;
}
