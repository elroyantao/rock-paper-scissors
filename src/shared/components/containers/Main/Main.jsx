import React, { Component } from 'react'

import Controls from '../../containers/Controls/Controls'
import Score from '../../presentation/Score/Score'
import styles from './Main.css'

export default class Main extends Component {
  render() {
    return (
      <div className="Main">
        <Score />
        <Controls />
      </div>
    )
  }
}
