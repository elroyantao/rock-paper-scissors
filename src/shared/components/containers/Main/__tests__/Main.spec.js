import testComponentHelper from '../../../../lib/test-helper'
import Main from '../Main'

describe('<Main /> ', () => {
  const initialState = {}
  const renderComponent = testComponentHelper(Main)
  describe('@renders', () => {
    it('in default state', () => {
      expect(renderComponent(initialState).getTree())
        .toMatchSnapshot()
    })
  })
})
