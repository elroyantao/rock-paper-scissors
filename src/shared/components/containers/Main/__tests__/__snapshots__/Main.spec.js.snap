// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main />  @renders in default state 1`] = `
<div
  className="Main"
>
  <Connect(Score) />
  <Connect(Controls) />
</div>
`;
